### PR TITLE
Add collector component

### PR DESCRIPTION
**Pull request by kickoke**
### Problem
We wanted to reuse a MDX file that had a variable. 
For some reasons (@tkatsoulas feel free to add them here) we weren't able to make it fully work. 

After countless hours of trying to implement MDX correctly (😭  mishmash of Markdown, Javascript, and HTML), I simply resorted to writing a JS component. ...Yes, we 'lose' the benefit of writing markdown in this particular instance, but taking 5 mins to assign tags to style the output is way better than wasting 4 hours of making MDX work (RIP brain). 

### Solution

I have written a simple JSX function that takes an argument, in this case the collector, and passes it to the respective variables to create dynamic content. 
I strongly suggest that for components that need a variable, we just resort to JS the next time. 
For components that don't need to be adapted we can simply use markdown. 

Closes #896 

🎤  drop. 

Todo: 
- [ ] Clean up old branches

### CODE REVIEW

**Comment by kickoke** [comment]
I will structure the ActiveMQ doc according to the [template](https://netdata.atlassian.net/wiki/spaces/DT/pages/1397161985/Topic+types+and+templates#Collector-reference-template) and then we can merge this beautiful baby :)

**Comment by kickoke** [comment]
Short summary from the meeting: 
Ilya is fine with our approach and supports our use of components! :) 
We should include a note in files that contain components that will be rendered by Docusaurus and have an ingest rule that filters out that note for learn.

File: src/components/Collectors/_collector-config.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+/>
+
+configURL: Assign the Github URL of the collector's configuration file
+moduleName: Assign the name of the collector in the following schema: PLUGIN/COLLECTOR.conf */
+}
+
+import React from 'react';
+import CodeBlock from '@theme/CodeBlock';
+
+export default function CollectorConfiguration({ configURL, moduleName }) {
```
Comment (kickoke) [inline src/components/Collectors/_collector-config.jsx:19]:
```suggestion
export default function EnableCollector({ configURL, moduleName }) {
```

**Review by kickoke — COMMENTED** [review]
Merge the collector components all into a single file.

**Review by DShreve2 — APPROVED** [review]

**Pull request merged**